datasource/ethereum/src/block_ingestor.rs

```diff
@@ -60,10 +60,22 @@ where
                 static_self.do_poll().then(move |result| {
                     if let Err(e) = result {
                         // Some polls will fail due to transient issues
-                        warn!(
-                            static_self.logger,
-                            "Trying again after block polling failed: {:?}", e
-                        );
+                        match e {
+                            BlockIngestorError::BlockUnavailable(block_hash) => {
+                                trace!(
+                                    static_self.logger,
+                                    "Trying again after block polling failed: \
+                                    block data unavailable, block was likely uncled";
+                                    "block_hash" => format!("{:?}", block_hash)
```

Comment:
This message now duplicates the `Fail` derivation, you can just print `e` instead.